newIDE/app/src/SceneEditor/index.js

```diff
@@ -205,6 +205,12 @@ export default class SceneEditor extends Component {
     this.setState({ scenePropertiesDialogOpen: open });
   };
 
+  openObjectEditor = () => {
+    if(!this.instancesSelection.getSelectedInstances()[0]) {return};
```

Comment:
nitpicking: You can use `if (!this.instanceSelection.hasSelectedInstances())`.

Reply:
(it's slightly more readable and will work if later the internal implementation of InstancesSelection is changed to use something else than an array).